test/test.faultydriver.html

```diff
@@ -25,6 +25,9 @@
     <!-- Modernizr  -->
     <script src="/bower_components/modernizr/modernizr.js"></script>
 
+    <!-- Promise polyfill -->
+    <script src="/bower_components/es6-promise/promise.js"></script>
+
```

Comment:
I couldn't figure out how to get `window.require('promise')` to work as it was working in the old tests, so I figured it was easier to just drop in a promise polyfill for the tests. Notice I only inserted them where the tests themselves relied on `window.Promise`; `lie` itself only relies on `window.Promise` in `utils/promise.js`, where it prefers the global Promise and then falls back to `lie` (or falls back to nothing, in the case of `nopromises`).